x-pack/test/apm_api_integration/basic/tests/transaction_groups/distribution.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+import qs from 'querystring';
+import { isEmpty } from 'lodash';
+import archives_metadata from '../../../common/archives_metadata';
+import { expectSnapshot } from '../../../common/match_snapshot';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+
+  const archiveName = 'apm_8.0.0';
+  const metadata = archives_metadata[archiveName];
+
+  // url parameters
+  const { start, end } = metadata;
+  const uiFilters = {};
+
+  const url = `/api/apm/services/opbeans-java/transaction_groups/distribution?${qs.stringify({
+    start,
+    end,
+    uiFilters,
```

Comment:
Tiny nit: this might as well be inlined if we don't use `uiFilters` elsewhere (similar to `transactionName` and `transactionType`)

```suggestion
    uiFilters: {},
```

Reply:
cheers, nice little improvement. (will do it for start/end as well).